var/spack/repos/builtin/packages/doxygen/package.py

```diff
@@ -50,7 +50,7 @@ class Doxygen(Package):
     depends_on("graphviz", when="+graphviz", type='run')
 
     def install(self, spec, prefix):
-        cmake('.', *std_cmake_args)
```

Comment:
i think you know the answer already https://groups.google.com/forum/?fromgroups#!topic/spack/gxizu9g40oU :smile: